contracts/WrappedTokenBridgeERC20Upgradable.sol

```diff
@@ -0,0 +1,35 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.17;
+
+import {LzLib} from "@layerzerolabs/solidity-examples/contracts/libraries/LzLib.sol";
+import {WrappedTokenBridgeUpgradable} from "./WrappedTokenBridgeUpgradable.sol";
+import {IBridgedERC20} from "./interfaces/IBridgedERC20.sol";
+
+/// @dev Mints a wrapped token when a message received from a remote chain and burns a wrapped token when bridging to a remote chain
+contract WrappedTokenBridgeERC20Upgradable is WrappedTokenBridgeUpgradable {
+    /// @notice Bridges `localToken` to the remote chain
+    /// @dev Burns wrapped tokens and sends LZ message to the remote chain to unlock original tokens
+    function bridge(address localToken, uint16 remoteChainId, uint amount, address to, bool unwrapWeth, LzLib.CallParams calldata callParams, bytes memory adapterParams) external payable override nonReentrant whenNotPaused {
+        require(localToken != address(0), "WrappedTokenBridge: invalid token");
+        require(to != address(0), "WrappedTokenBridge: invalid to");
+        require(amount > 0, "WrappedTokenBridge: invalid amount");
+        _checkAdapterParams(remoteChainId, PT_UNLOCK, adapterParams);
+
+        address remoteToken = localToRemote[localToken][remoteChainId];
+        require(remoteToken != address(0), "WrappedTokenBridge: token is not supported");
+        require(totalValueLocked[remoteChainId][remoteToken] >= amount, "WrappedTokenBridge: insufficient liquidity on the destination");
+
+        totalValueLocked[remoteChainId][remoteToken] -= amount;
+        IBridgedERC20(localToken).burnFrom(msg.sender, amount);
```

Comment:
That's the only change here, right @MayankMittal1 ?

Reply:
@leonprou Yes